modules/dynamic/src/smithy4s/dynamic/DynamicModelCompiler.scala

```diff
@@ -61,7 +64,12 @@ object Compiler {
     }
 
     def toHint(id: ShapeId, tr: Document): Option[Hint] = {
-      hintsMap.get(id).flatMap(toHintAux(_, tr))
+      // todo: this is an ugly hack, get rid of it
```

Comment:
This is to work around the fact that `smithy.api.http` in the model doesn't match `smithy.api.Http` in the known trait, which is used by all the interpreters and such

Reply:
Oh, it's a rendering problem, I uppercase too eagerly and should keep the original name around for longer